subjects/bowling_score/README.md

```diff
@@ -0,0 +1,106 @@
+## bowling_score
+
+### Instructions
+
+Create a library that implements a `struct` to calculate the score of a player in the game of [Bowling](https://en.wikipedia.org/wiki/Bowling).
+
+The methods of `BowlingGame` will be:
+
+- `roll`: It will take the number of knocked pins and return `Ok(())` if the roll is valid or an `Err(Error::...)` if the roll is invalid.
```

Comment:
"It will take the number of pins knocked down and return"

Reply:
done